Core/Loaders.cpp

```diff
@@ -36,7 +36,7 @@
 #include "Core/ELF/ParamSFO.h"
 
 FileLoader *ConstructFileLoader(const std::string &filename) {
-	if (filename.find("http://") == 0 || filename.find("https://") == 0)
+	if (filename.compare(0, 7, "http://") == 0 || filename.compare(0, 8, "https://") == 0)
```

Comment:
I don't like adding magic numbers for no practical gain.  It just means someone will copy and paste another protocol in some day and it won't work because they forgot to change the `7` that they didn't realize had to be the length of the component.

I also don't think `filename.compare(0, strlen("http://"), "http://")` is worth doing, because I'm not a member of the We Enjoy Typing brigade.

I think we have a `startsWith` utility function somewhere if you feel like it really needs to change.

-[Unknown]